changelogs/fragments/64-use-proxysql_common_argument_spec.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- use ``proxysql_common_argument_spec`` in all modules (https://github.com/ansible-collections/community.proxysql/pull/64).
```

```suggestion
minor_changes:
- use ``proxysql_common_argument_spec`` in all modules (https://github.com/ansible-collections/community.proxysql/pull/64).
```
I'm not sure where we need a fragment for this as it doesn't touch user experience. It could be potentially helpful for developers and in this case I'd add something like `code refactoring` or we can remove the fragment entirely.